src/main/resources/application.yml

```diff
@@ -9,12 +9,15 @@ security:
   jwt:
     issuer: ${jwt.issuer:CodeSupport.dev}
     expiration: ${jwt.expiration:10m}
+  discord:
```

Comment:
Are all the values for the keys in `discord` meant to have colons at the end?

Reply:
If you mean the ones like `discord:` that's required yaml syntax

Reply:
I meant before the `}`

Reply:
Defaults it to a blank string, not necessary for logic, but makes a more clear error message later if not set